### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Third-party applications can include their configuration tools in
 "Configuration Center", too. This applies e. g. to "Connman UI Setup" of
 [cmst](https://github.com/andrew-bibb/cmst) or "SDDM Configuration" of
 [sddm-config-editor](https://github.com/hagabaka/sddm-config-editor).
+Not already included settings are shown under "Other Settings".
```
Comment (tsujan) [inline README.md:79]: Maybe: Other kinds of settings are shown under "Other Settings".

Comment (stefonarch) [reply README.md:79]: Actually I don't remember exactly the mechanism. "Other kind of settings" is not very clear about how it works.

Comment (tsujan) [reply README.md:79]: > Actually I don't remember exactly the mechanism.

It includes desktop files whose `Categories` contain `Settings` but don't belong to the two other categories.

But I just meant the English; it sounded too German ;)

**Review by tsujan — APPROVED** [review]
GTM

**Pull request merged**